src/client/pages/_slug.vue

```diff
@@ -37,31 +37,31 @@ export default {
 @import '../components/app-core/index.css';
 
 .page__content {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  max-width: 920px;
+  margin: 0 auto;
   padding: 0 var(--spacing-default);
+  max-width: 920px;
 }
 
-.page__content__sidebar {
+.page__sidebar {
+  margin: 0 auto var(--spacing-double) auto;
   width: 100%;
-  margin-bottom: var(--spacing-double);
+  max-width: var(--max-width);
 }
 
 @media screen and (min-width: 880px) {
   .page__content {
-    flex-direction: row;
+    display: flex;
     justify-content: space-between;
     align-items: flex-start;
-    margin: 0 auto;
-    padding: 0 var(--spacing-default);
+    padding: 0 var(--spacing-double);
   }
 
-  .page__content__sidebar {
+  .page__sidebar {
+    position: sticky;
+    top: 80px;
```

Comment:
Since the sticky position of the side bar is based on the height of the `app header` I think you should use the variable `--app-header-height` here. And it would be nice if there is a little spacing between the header and the ToC, so I would suggest `top: calc(var(--app-header-height) + var(--spacing-default));`.